JSONCasing appears to be a git repo or submodule

**Test case**

Create a new folder and add a package.json:

```json
{
  "name": "json-casing-test",
  "version": "1.0.0",
  "dependencies": {
    "json-casing": "^1.1.2"
  }
}
```

Run `npm install`.
Next, run `npm install` again. 
You get the following error:

```bash
npm ERR! path /json-casing-test/node_modules/json-casing
npm ERR! code EISGIT
npm ERR! git /json-casing-test/node_modules/json-casing: Appears to be a git repo or submodule.
npm ERR! git     /json-casing-test/node_modules/json-casing
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

**Cause**

The published package contains a *.git* folder. You can verify this by running [`npm pack`](https://docs.npmjs.com/cli/pack) and checking the content of the generated tarball.

**Solution**

Remove all unnecessary files (including the *.git* folder) from the published package. You can do this by using package.json's `files` array or *.npmignore*

--- a/package.json
+++ b/package.json
@@ -15,7 +15,9 @@
   "scripts": {
     "test": "jest --coverage",
     "tdd": "jest --watch",
-    "build": "npm run test && webpack"
+    "build": "npm run test && webpack",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm run build"
   },
   "author": "Stewart Anderson",
   "license": "GPL-3.0",
@@ -31,5 +33,8 @@
   },
   "dependencies": {
     "babel-runtime": "^6.26.0"
-  }
+  },
+  "files": [
+    "build"
+  ]
 }

